Gemfile.lock

```diff
@@ -93,40 +100,57 @@ GEM
       railties (>= 4.1.0)
       responders
       warden (~> 1.2.3)
+    digest (3.1.0)
     erubi (1.10.0)
     execjs (2.8.1)
     ffi (1.15.4)
-    globalid (0.5.2)
+    globalid (1.0.0)
       activesupport (>= 5.0)
-    i18n (1.8.10)
+    i18n (1.8.11)
       concurrent-ruby (~> 1.0)
     importmap-rails (0.7.4)
       rails (>= 6.0.0)
     io-console (0.5.9)
+    io-wait (0.2.1)
     irb (1.3.7)
       reline (>= 0.2.7)
     jbuilder (2.11.2)
       activesupport (>= 5.0.0)
     jsbundling-rails (0.1.9)
       railties (>= 6.0.0)
-    loofah (2.12.0)
+    loofah (2.13.0)
```

Comment:
I'm not sure how big of a deal this security vulnerability is. Since it's asking to "upgrade" back down to 2.2.1 and we're currently using 2.12.0, is going up to 2.13.0 going to make things any worse than they already are?